content/sensu-go/6.5/sensuctl/environment-variables.md

```diff
@@ -12,20 +11,72 @@ menu:
     parent: sensuctl
 ---
 
-Sensuctl includes the `sensuctl env` command to help export and set environment variables on your systems.
+Sensu allows you to set sensuctl environment variables for a [single sensuctl command][1], with [sensuctl configure][2], or with [sensuctl env][3].
```

Comment:
Also, the first thing I wondered when trying to sort this out is why the `sensuctl env` env var options are different from the env var options for the other two methods. Is there a reason we can provide to help prevent confusion?

Reply:
The environment variables are directly taken from the equivalent flags. I don't know how the variables were selected when `sensuctl env` was created. Again I'm sure @calebhailey or @portertech has more insights.